User/InMemoryUserRepository.php

```diff
@@ -54,6 +54,18 @@ public function findByUsername($username)
     /**
      * {@inheritDoc}
      */
+    public function find($id)
+    {
+        foreach ($this->users as $user) {
+            if ($user->getId() === $id) {
+                return $user;
+            }
+        }
```

Comment:
returns null when not found http://www.doctrine-project.org/api/orm/2.2/source-class-Doctrine.ORM.EntityRepository.html#100-147